src/IECoreScene/MeshPrimitiveEvaluator.cpp

```diff
@@ -986,7 +1064,7 @@ bool MeshPrimitiveEvaluator::pointAtUVWalk( UVBoundTree::NodeIndex nodeIndex, co
 			Imath::V2f uv[3];
 			triangleUVs( triangleIndex, vertexIds, uv );
 
-			if( triangleContainsPoint( uv[0], uv[1], uv[2], targetUV, result->m_bary ) )
+			if( triangleContainsPointV2f( uv[0], uv[1], uv[2], targetUV, result->m_bary ) )
```

Comment:
Is it a concern that we are changing behaviour at interior edges, due to points within the tolerance matching whichever triangle gets tested first?  It does seem like this could result in worse behaviour in some cases?  If we wanted to address this, I guess we would need triangleContainsPointV2f to have 3 return values, for Inside, Outside, and Boundary, and we would only return immediately if we find a triangle where we are unambiguously inside, otherwise we would test all triangles in the current UVTree node before returning a boundary hit?